js/settingsHelpers.js

```diff
@@ -650,6 +696,23 @@ export const unlinkFirebaseAuthProvider = async (providerType, userData) => {
   }
 };
 
+export const updateToNoReplyEmail = async (uid, noReplyEmail) => {
+    document.getElementById('loginUpdateFail').style.display = 'none';
+    document.getElementById('loginUpdateSuccess').style.display = 'none';
+    
+    const valuesForFirebaseAuth = {};
+    valuesForFirebaseAuth['email'] = noReplyEmail;
+    valuesForFirebaseAuth['uid'] = uid;
```

Comment:
Is `valuesForFirebaseAuth` used? If not, it's not needed.

Reply:
Great catch. Not used here. Removed.